src/com/dynatrace/loadrunner/CommandLineParser.java

```diff
@@ -0,0 +1,169 @@
+package com.dynatrace.loadrunner;
+
+import java.util.HashMap;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class CommandLineParser {
+
+	public HashMap<String, String> arguments;
+
+	public CommandLineParser(String[] args) {
+
+		arguments = new HashMap<String, String>();
+
+		String key = null;
+		String value = null;
+		int index = 0;
+
+		for (int i = 0; i < args.length; i++) {
+
+			if (args[i].startsWith("--")) {
```

Comment:
is it documented that we accept params with one or two colons?
If it is, then:
```
if(args[i].startsWith("-")) {
  key = args[i].substring(args[i].lastIndexOf("-"); // or so
}
```
If not - leave only one condition.